### PR TITLE
changes for 0.12

### PR DESCRIPTION
**Pull request by justinwoo**

### CODE REVIEW

**Comment by justinwoo** [comment]
finally updated this PR, should be good to go

File: package.json [hunk 1]
```diff
@@ -6,9 +6,8 @@
     "test": "pulp test"
   },
   "devDependencies": {
-    "pulp": "^11.0.2",
-    "purescript-psa": "^0.5.1",
-    "purescript": "^0.11.6",
```
Comment (safareli) [inline package.json]: I guess version could be just bumped to `^0.12.0` here too

Comment (cryogenian) [reply package.json]: Nope, at least for me `purescript` package doesn't work, I use `slamdata/node-purescript#0.12`

**Comment by cryogenian** [comment]
@justinwoo Thank you very much!

**Pull request merged**